superchain/configs/sepolia/ink.toml

```diff
@@ -60,10 +60,10 @@ max_sequencer_drift = 600
   SystemConfigProxy = "0x05C993e60179f28bF649a2Bb5b00b5F4283bD525"
   ProxyAdmin = "0xd7dB319a49362b2328cf417a934300cCcB442C8d"
   SuperchainConfig = "0xC2Be75506d5724086DEB7245bd260Cc9753911Be"
-  AnchorStateRegistryProxy = "0xDa9916204568e2A8d689f775747D9e7FE17F7560"
-  DelayedWETHProxy = "0x237840A6Bfd822039d9cC00e1E7BAE280d4F2D49"
+  AnchorStateRegistryProxy = "0x299D7Ea9f0B584cfaF2a5341D151B44967594cA9"
+  DelayedWETHProxy = "0x1E4578692cD39faAD678F5D78B65B584B355Bc69"
```

Comment:
doesn't match this value: https://github.com/ethereum-optimism/superchain-registry/blob/main/superchain/extra/addresses/addresses.json#L1109

Reply:
9a7dcf1